UnityProject/Assets/StreamingAssets/Maps/MainStations/BoxStationV1.json

```diff
@@ -362183,6 +362183,12 @@
               ]
             }
           },
+          {
+            "GitID": "CD2uUGXjPEuvXGheerXA_0┼0┼0┼",
+            "PrefabID": "CD2uUGXjPEuvXGheerXA",
+            "Name": "MainStation",
```

Comment:
is this the name of the gameobject that has the marker component? Can it be something else than ``MainStation``?

Reply:
yeah, just couldn't think of a good name for marking what  makes the main sstation

Reply:
yeah but does it have to be that or could the ``Name`` field have the actual name of the station? We need a field with the name of the station because this is not good 
![image](https://github.com/user-attachments/assets/ed58e0ce-6979-444a-85b5-49eecac5e040)

Reply:
the issue as we have to re-factor it since it was loading by scene name, Would have to set up the thing to get the display name instead of the filename